examples/analytics.amp.html

```diff
@@ -442,6 +442,39 @@
 </script>
 </amp-analytics>
 <!-- End SOASTA mPulse example -->
+
+<!-- Adobe Analytics example -->
+<!--
+For complete documentation and additional examples please see: https://marketing.adobe.com/resources/help/en_US/sc/implement/accelerated-mobile-pages.html
+-->
+<amp-analytics type=adobeanalytics>
+<script type="application/json">
+{
+  "requests": {
```

Comment:
this looks good but just wanted to point out that we can do this using extraUrlParams. I just sent out a PR for the piece that was missing. #3168